nginx_data/trusted.html

```diff
@@ -1,4 +1,4 @@
 <html>
         <H1>Welcome to the Trusted Site</H1>
-        <H2>This is secured by a private TLS Cert</H2>
+        <H2>This is secured using TLS Mutual Authentication</H2>
```

Comment:
Don't we want two sites? One with client certs and one without?

I would also recommend a third. One with optional client certificates.